app/src/main/res/values/preferences.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="default_fragment_recording_seconds">5</integer>
+    <string name="fragment_recording_seconds">fragment_recording_seconds</string>
```

Comment:
why not in strings.xml? The other preference id's are also there.

Reply:
I don't like to put preferences that I use in a strings XML file that I use for strings

Reply:
I can do that in a next "chore" pr.